index.html

```diff
@@ -353,8 +356,38 @@ <h1 class="msg2">About <br> Myself</h1>
            </div>
           </section>
 
- </div>
 
+          <section class="form-section">
+            <div class="greet" font-weight="bold">
+                <h1 class="msg">Contact me</h1> 
+            </div>
+            <p class="info">If you have an application you are interested in developing,
+                          a feature that you need built or a project that needs coding.
+                          I’d love to help with it </p>
+        
+            <div class="form">
```

Comment:
- [ ] I think it would be better if you use here `form` tag because you want to add action and method here, as I can see you didn't use the formspree so please read more so after submitting the data it will be inside formspree so kindly read more and update the form to make it work preoplry 

![image](https://user-images.githubusercontent.com/58553711/199238137-8fb7a0f4-45f7-4c1e-b3ef-42ab24d160d1.png)


- Your final code will be something like this and you will make a link for you to put it in actions="" so you will sign up and you will put your link there 🔥💯🚀

![image](https://user-images.githubusercontent.com/58553711/199238281-f8f2ac1e-b223-4cbe-bd79-450fd2b0fb41.png)